code/modules/research/chem_consumer.dm

```diff
@@ -0,0 +1,134 @@
+// 100 research points for 1 unit of the output chem. Seems balanced considering each node is 2500 points
+// An entire beaker of something can speed unlock 4 entire nodes
+#define RESEARCHPOINTS_PER_UNIT 100
+
+/obj/machinery/research_chem_consumer
+	name = "research chemical consumer"
+	desc = "Consumes chemicals to gain research points"
+	icon_state = "chemical_idle"
+	icon = 'icons/obj/machines/research_consumers.dmi'
+	anchored = TRUE
+	density = TRUE
+	var/obj/item/reagent_containers/beaker
+
+/obj/machinery/research_chem_consumer/Initialize(mapload)
+	. = ..()
+	component_parts = list()
+	component_parts += new /obj/item/circuitboard/research_chem_consumer(null)
+
+/obj/machinery/research_chem_consumer/update_icon()
+	if(panel_open)
+		icon_state = "chemical_open"
+	else if(stat & NOPOWER)
+		icon_state = "chemical_powerless"
+	else if(istype(beaker))
```

Comment:
Do you need the `istype()` here and on [L101](https://github.com/ParadiseSS13/Paradise/blob/fa553e08612a6f9729aa8da1078bd2c8894936f3/code/modules/research/chem_consumer.dm#L101)? It seems like if there's something inserted it'll always be a beaker.
```suggestion
	else if(beaker)
```